docs/faq/get_queue.md

```diff
@@ -0,0 +1,28 @@
+# MA get_queue Action
+
+This action allows you to retrieve the full details of a queue
+
+![image](../assets/screenshots/service-call/get_queue.png)
+
+The Queue ID can be found by inspecting the MA Player entity and finding the value for `active_queue`
```

Comment:
This can be dropped now that its become an entity service. It will now just auto pick the currently active queue for a player (if any!)

Reply:
Fixed. Text removed and new image uploaded.